src/types/object.js

```diff
@@ -2,6 +2,7 @@ import { append, filter, map } from 'funcadelic';
 import parameterized from '../parameterized';
 import { mount, valueOf } from '../meta';
 import { create } from '../microstates';
+import { Entry } from '../query';
```

Comment:
`Entry` is a concept of `Object`, and not `Query`, and so should probably reside in the object class.